src/service/platform-impl.js

```diff
@@ -79,7 +79,8 @@ export class Platform {
    */
   isIe() {
     return (/MSIE/i.test(this.navigator_.userAgent) ||
```

Comment:
This is causing a circular-dep because we're importing `polyfills` in `service.js` (or at least this is what I think the following error is)

```
src/service/platform-impl.js:17: ERROR - required "module$src$service" namespace not provided yet
import {fromClass} from '../service';
```

@camelburrito suggested we move it to a `user-agent.js` file and import it into both of them. You're ok with that?